ui/javascripts/app/controllers.js

```diff
@@ -58,12 +59,21 @@ App.DcController = Ember.Controller.extend({
   // Boolean if the datacenter has any failing checks.
   //
   hasFailingChecks: Ember.computed.gt('totalChecksFailing', 0),
-
+  
+  init: function() {
+    if(App.get('settings.v1-notification-hidden', true)) {
+      this.set('isNotificationVisible', false);
+    }
+  },
   actions: {
     // Hide and show the dropdown menu
     toggle: function(item){
       this.toggleProperty('isDropdownVisible');
     },
+    hideNotification: function(e) {
+      App.set('settings.v1-notification-hidden', true);
```

Comment:
So this will persist in local storage or something? If they blow away the cache they'll have to x out but otherwise not?

Reply:
Ha nvm ready your PR description after 😂

Reply:
Yeah, I was going to use a cookie, but the UI is using localStorage so I stuck to that, if someone blows that away, then they'll get the notification again.